packages/sdk/source/services/shared.contract.ts

```diff
@@ -13,4 +13,5 @@ export interface IdentityOptions {
 	bip49?: IdentityLevels;
 	bip84?: IdentityLevels;
 	multiSignature?: MultiSignatureAsset;
+	locale?: string;
```

Comment:
```suggestion
	bip39Locale?: string;
```

Also move it right after `bip39` to keep it organised.

Reply:
Patched.